README.md

```diff
@@ -115,6 +116,58 @@ proxy.
   script. Hit the Connect button, enter a password if the VNC server has one
   configured, and enjoy!
 
+### Installation from Snap Package
+To install from Snap:
+
+`sudo snap install novnc`
```

Comment:
I'm a bit cautious about this. The rest of the README is for running your current working copy. Doing an install like this would mean running the latest stable, not what you have right now.

What is the process for building your own snap package and using that? Could we have that, and suggest this line if you just want the latest stable version?

Reply:
I'm not really sure what you mean here - the Snap package isn't meant to be for users to build*. The idea is that by adding in the snap/ directory (metadata/scripts for the snap package itself) plus the small changes I had to make to stuff under utils/ you have everything needed to submit the Snap package to Canonical's Snap store. Then anyone can install noVNC with just a 'snap install novnc'. There shouldn't be any major changes needed to the stuff under snap/ as you make further releases of noVNC

* technically you, or anyone else, can build your own copy of the Snap package and use it like I have been for testing. Note that you have to use the --dangerous switch to install it, this is something Canonical deliberately put in Snap to encourage people not to build and host their own .snap packages but to submit them to the store. Build instructions:

- git clone https://github.com/tkedwards/noVNC.git # the correct URL will be https://github.com/novnc/noVNC.git once everything is merged in
- cd noVNC/
- git checkout snap_package
- edit parts-novnc-source in snap/snapcraft.yaml to specify https://github.com/tkedwards/noVNC.git as the source (this step shouldn't be necessary once everything's merged in)
- also in snap/snapcraft.yaml add source-branch: snap_package (this shouldn't be necessary once this is all merged)
- snapcraft snap # add --use-lxd if on hardware which doesn't have the VT-x virtualisation extensions (e.g. a VPS)
- install the snap with sudo snap install --dangerous novnc_1.1.0_amd64.snap

Reply:
> I'm not really sure what you mean here - the Snap package isn't meant to be for users to build*.

Most of the time that is indeed true. The problem is that right now `README.md` describes how to run *this specific* copy of noVNC. How to run the latest stable is currently something we only refer to on the home page.

Perhaps we need to have an "Installation" section in the README as well. Or if it is sufficient to document the snap stuff on just the home page.

Reply:
Yes, it would be good to have those instructions on the home page (after these changes have been merged if you approve and the novnc Snap submitted to Canonical's Snap 'store').

Reply:
Hi, 

So is there something else I should add to the README in this pull request?

Reply:
You should clarify in the README that it's the latest stable version they will install, just something simple like this:

> Running the command below will install the latest release of noVNC from Snap:

Reply:
All good, I've updated the README, let me know if anything else

Reply:
Great! Thank you @tkedwards